### DISCUSSION TITLE
FallingBlock Entity is stacking

### DISCUSSION THREAD

**Discussion by phoenix3394** [Minestom/Minestom#1036] [Help]
Hey, I made a method that spawn a fallingblock when nothing is below the block (Gravity for Sand, etc) but after a while the Fallingblock entites are stacking and remove only at the end (see clip). How to fix it? Please help me

https://user-images.githubusercontent.com/102826375/167130114-b1f96021-5b47-4544-bc50-7a7180fb1d42.mp4

@TheMode

**Comment by KrystilizeNevaDies** [comment]
This is because of client-side prediction.
Minestom doesn't remove falling block entities just because they are on top of a torch.

So the client thinks that the entities should be removed, but we know they shouldn't, and after a couple seconds the client realises that the entities are still there.

**Comment by phoenix3394** [comment]
![image](https://user-images.githubusercontent.com/102826375/167263378-ffda3e41-d382-4328-8898-dbf8e9a031ab.png)

I have created this method that remove the falling block entity. In the first few secounds it works but then it starts stacking

This method got execute every tick
